### PR TITLE
osm-seed tiler containers

### PR DESCRIPTION
**Pull request by Rub21**
cc. @batpad

### CODE REVIEW

File: helm/osm-seed/templates/NOTES.txt [hunk 1]
```diff
@@ -17,8 +17,4 @@
            You can watch the status of by running 'kubectl get svc -w {{ template "osm-seed.fullname" . }}'
   export SERVICE_IP=$(kubectl get svc --namespace {{ .Release.Namespace }} {{ template "osm-seed.fullname" . }}-web -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
   echo http://$SERVICE_IP
```
Comment (batpad) [inline helm/osm-seed/templates/NOTES.txt:19]: Nice catch! Thanks for making this easier to read @Rub21

File: helm/osm-seed/values.yaml [hunk 2]
```diff
@@ -4,18 +4,21 @@
 # ====================================================================================================
 # ====================================================================================================
 # The version of the image group in osm-seed, get it here: https://hub.docker.com/r/developmentseed/osmseed-web/tags/
-osmSeedVersion: cloudProvider-b1fee6e
+osmSeedVersion: tiler-6be452f
```
Comment (batpad) [inline helm/osm-seed/values.yaml:7]: @Rub21 can we try and avoid making these kinds of changes in the default values file, and leave default pointed to `master` or so? And just over-ride locally?

**Review by batpad — APPROVED** [review]
@Rub21 this looks really good! Left one really minor comment.

One question: right now there are separate variables for `tilerImposm`, `tilerVisor`, etc. - does it make sense to make them a single variable like `tileServer` which when set to `true` enables all these things? However, let's not worry about that now - just making a note for the future.

Also, the circle config.yml is growing a bit out of hand - can create a separate ticket to look into cleaning that up some-how, but can deal with that separately.

Nice work @Rub21 !

File: helm/osm-seed/values.yaml (hunk 2 above)
Comment (Rub21) [reply helm/osm-seed/values.yaml:7]: @batpad , ok, once I merge these changes it will generate a new one and I will update with that value..

**Pull request merged**